dockstore-integration-testing/src/test/java/io/dockstore/client/cli/BasicET.java

```diff
@@ -0,0 +1,444 @@
+ * @author aduncan
+ */
+public class BasicET {
+
+        @ClassRule
+        public static final DropwizardAppRule<DockstoreWebserviceConfiguration> RULE = new DropwizardAppRule<>(
+                DockstoreWebserviceApplication.class, ResourceHelpers.resourceFilePath("dockstoreTest.yml"));
+
+        @Rule
+        public final ExpectedSystemExit systemExit = ExpectedSystemExit.none();
+
+        @Before
+        public void clearDBandSetup() throws IOException, TimeoutException {
+                clearStateMakePrivate();
+                Client.main(new String[] { "--config", ResourceHelpers.resourceFilePath("config_file.txt"), "refresh" });
+        }
+
+        /*
+         General tests -
+         Not based on Quay/Dockerhub or Github/Bitbucket
+          */
+        /**
+         * Checks that all automatic containers have been found by dockstore and are not registered/published
+         */
+        @Test
+        public void testListAvailableContainers() {
+                final CommonTestUtilities.TestingPostgres testingPostgres = getTestingPostgres();
+                final long count = testingPostgres.runSelectStatement("select count(*) from container where isregistered='f'", new ScalarHandler<>());
+                Assert.assertTrue("there should be 4 entries", count == 4);
+        }
+
+        /**
+         * Will test adding/editing/deleting container related labels (for search)
+         */
+        @Ignore
+        public void testAddEditRemoveLabel() {
+                // Todo : test adding/editing/removing labels to a container (should be available to search)
+        }
+
+        /**
+         * Will test altering the cwl and dockerfile paths to valid and invalid locations
+         */
+        @Ignore
+        public void testVersionTagCWLAndDockerfilePathsAlteration() {
+                // Todo : test editing cwl path (valid and invalid) and  dockerfile path (valid and invalid)
+        }
+
+        /**
+         * Will test adding version tags to a manually registered container
+         */
+        @Ignore
+        public void testAddVersionTagManualContainer() {
+                // Todo : test adding version tags to a manually added container
+        }
+
+        /**
+         * Will test hiding and unhiding different versions of a container
+         */
+        @Ignore
+        public void testVersionTagHide() {
+                // Todo : test hiding and unhiding different versions of a container
+        }
+
+        /*
+         Test Quay and Github -
+         These tests are focused on testing containers created from Quay and Github repositories
+          */
+        /**
+         * Checks that the two Quay/Github containers were automatically found
+         */
+        @Test
+        public void testQuayGithubAutoRegistration(){
+                final CommonTestUtilities.TestingPostgres testingPostgres = getTestingPostgres();
```

Comment:
This is due to a refresh occurring in the Before function clearDBandSetup(). This function clears the database, inserts user information and tokens, and then a refresh is called in order to populate the database with the containers.  I thought the refresh would be ok to put here since it is necessary for most of the tests and if a Quay token exists in the database for a user then the automatic Quay containers would exist in the database already.